### PR TITLE
Standardize variadic passing of groups/permissions

### PR DESCRIPTION
**Pull request by lonnieezell**
Needs to rebase after merging #255 

- Standardize variadic passing of groups/permissions to the syncGroups and syncPermissions methods. 
- Added docs for groups to the quick start.

### CODE REVIEW

**Review by MGatner — APPROVED** [review]
I like the change to variadic with types. I can't tell what changed in the quick start until this is rebased after the initial quick start PR.

**Review by kenjis — CHANGES_REQUESTED** [review]
#255 should be merged first, and
`docs/2 - quickstart.md` should be renamed.

File: docs/2 - quickstart.md [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+public array $matrix = [
+    'superadmin' => [
+        'admin.*',
+        'users.*',
+        'beta.access',
+    ],
+    //
+];
+```
+
```
[the remark this answers is not in the document]
Comment (lonnieezell) [reply docs/2 - quickstart.md:158]: This is handled by the `can()` method below.

**Comment by lonnieezell** [comment]
Closing - new PR #266 is same thing, but cleaned up.

**Pull request closed without merging**